package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "heroku-slugs",
-  "version": "1.0.3",
+  "version": "1.0.4",
```

Comment:
Could you remove the upgrade from the PR? We'll do it ourselves after merging.

Reply:
Yeap, sorry. This was a work around for something I did locally.

Reply:
Ping :)